src/shogun/classifier/GaussianProcessClassification.h

```diff
@@ -80,23 +80,26 @@ class GaussianProcessClassification : public GaussianProcessMachine
 	 *
 	 * @return mean vector
 	 */
-	SGVector<float64_t> get_mean_vector(const std::shared_ptr<Features>& data);
+	SGVector<float64_t>
+	get_mean_vector(const std::shared_ptr<Features>& data) override;
```

Comment:
Do we really need this method btw? Because `apply` should already return the mean predictions, or?

Reply:
But doesn't this kind of go in hand with the variance?

Reply:
what I mean is that we would get the mean when predicting (what happens in classifier case actually?), but if we have `get_variance` the `get_mean` seems natural to have (imo).

Reply:
actually not necessarily, as in classification the posterior gp is not gaussian and the mean and variance vector will be those of the gaussian approximation (Laplace, variational, or EP ... all Gaussian). So let's keep it